TrenchBroom/TrenchBroom
# 4369: Introduce automatic dark mode (TrenchBroom Manual)

Resolves #4369

---

I've added a Dark Mode to the Reference Manual, making it easier to read at night. 

![image](https://github.com/TrenchBroom/TrenchBroom/assets/14064112/ccf321a9-beb7-4e6d-b323-0e22d152a257)

This mode changes automatically based on the computer's settings, thanks to a special feature in the coding. It tweaks some of the design elements without changing everything, keeping the overall look consistent.

![image](https://github.com/TrenchBroom/TrenchBroom/assets/14064112/e20e9887-a0a3-4193-9624-625ec020836c)

To test it:
- Switch to this new version by checking out the branch.
- Build the Reference Manual.
- Use your browser's tools to switch between Light and Dark Mode and see the changes.

If your computer is already set to Dark Mode, you'll see the changes right away. You can also switch using the native feature included in the DevTools of the browsers.

![image](https://github.com/TrenchBroom/TrenchBroom/assets/14064112/6f55f8b8-f576-486b-bb3c-becb96e5b913)

Plus, I've changed the main font to something that's easier on the eyes.

@kduske @ericwa Let me know if the colours are fine for you. I can change them easily if needed.
